src/domain/user.js

```diff
@@ -52,7 +52,7 @@ export default class User {
     bio,
     githubUrl,
     passwordHash = null,
-    role = 'STUDENT'
+    role
```

Comment:
Are you sure you want to remove the default role? 

What is the reasoning for doing this?